### DISCUSSION TITLE
How to customize colors in bootstrap 5?

### DISCUSSION THREAD

**Discussion by mrkhairullah** [twbs/bootstrap#34832] [Q&A]
Can someone help me learn how to customize colors in bootstrap 5? help

**Comment by adityadees** [comment]
What do you mean about customize colors?
external css or what?
maybe you can see this [color bootstrap](https://getbootstrap.com/docs/5.0/customize/color/#all-colors) and this [text-color](https://getbootstrap.com/docs/5.0/utilities/colors/)
